### PR TITLE
fix(nightly build): remove timestamp

### PR DESCRIPTION
**Pull request by TanklesXL**
removing the timestamp from the nightly build filenames

### CODE REVIEW

**Review by HarryET — COMMENTED** [review]
Looks good to me, the only thing I suggest if we aren't going to have it in the archive name or as the executable version we should edit the `Upload release archive` to update it's description. See docs here: https://github.com/softprops/action-gh-release#-customizing

Another comment, maybe it's worth keeping the executable version (in cargo.toml) to avoid conflicts, if we don't want date then maybe use short Git SHA?

File: .github/workflows/release-nightly.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
       - name: Update versions
         shell: bash
         run: |
-          version=0.0.0-nightly-$(date +"%y%m%d")
+          version=0.0.0-nightly
```
Comment (lpil) [inline .github/workflows/release-nightly.yaml:74]: Please revert to including the date in the version, so people can tell when they got their release. 🙏

Comment (TanklesXL) [reply .github/workflows/release-nightly.yaml:74]: Yeah that's what @HarryET mentioned as well, I was a little overzealous at first when I went about removing the date from everything 😆

Comment (TanklesXL) [reply .github/workflows/release-nightly.yaml]: Does this also apply to the change on line 164?

File: .github/workflows/release-nightly.yaml [hunk 2]
```diff
@@ -161,7 +164,7 @@ jobs:
...
```
Comment (HarryET) [inline .github/workflows/release-nightly.yaml:167]:
```suggestion
          version=0.0.0-nightly-$(date +"%y%m%d")
```
This line needs to be reverted as you suggested in a reply to @lpil's comment

**Review by HarryET — CHANGES_REQUESTED** [review]

**Review by lpil — APPROVED** [review]
Nice!

**Pull request merged**